src/pineko/evolve.py

```diff
@@ -153,6 +153,8 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
 
     with open(card_path, "w", encoding="UTF-8") as f:
         yaml.safe_dump(operators_card, f)
+        f.write(f"# {pineko_version=}")
```

Comment:
I guess this was not what you wanted to write, but:
```suggestion
        f.write(f"# pineko_version={pineko_version}")
```
(mainly because of the `=`, that makes little sense without `:` and a width)

Reply:
`{pineko_version=}` is shorthand for `pineko_version={pineko_version}`

But I can do `pineko_version: {pineko_version}` instead if you prefer the `:` (it's a comment so it doesn't really matter how is it written)

Reply:
Ah, great. I forgot about [this option](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals).

The reference to the `:` was only because `=` can be used as a format specifier, but the specifiers should be separated by the identifier with the `:`.

It is fine as it is.

Reply:
Btw, it was new in py3.8, that by now it is perfectly fine (py3.7 officially died one month ago).
https://docs.python.org/3/whatsnew/3.8.html#bpo-36817-whatsnew